GitCommands/Git/Commands.Arguments.cs

```diff
@@ -396,7 +396,7 @@ public static ArgumentString MergedBranches(bool includeRemote = false, bool ful
                 { fullRefname, @"--format=""%(refname)""" },
                 { includeRemote, "-a" },
                 "--merged",
-                commit
+                { !string.IsNullOrWhiteSpace(commit), commit.QuoteNE() }
```

Comment:
❔ There is no need for double check.
```suggestion
                { !string.IsNullOrWhiteSpace(commit), commit.Quote() }
```

Reply:
This is due to a test case with a ref with a space only  " ".
QuoteNE() will quote that and the test fails.
I did not want to implement a special handling just for this ref (other than the check).

That ref s illegal, so maybe remove the testcase instead.